OTAnalytics/plugin_video_processing/video_reader.py

```diff
@@ -41,4 +41,4 @@ def get_frame(self, video_path: Path, index: int) -> TrackImage:
         clip.close()
         if found is None:
             raise FrameDoesNotExistError(f"frame number '{index}' does not exist")
-        return PilImage(Image.fromarray(found))
+        return PilImage(Image.fromarray(found).convert("L"))
```

Comment:
Can we extract the literal `"L"` to a constant to make the code more readable?
As in,

```python
GRAYSCALE = "L"
return PilImage(Image.fromarray(found).convert(GRAYSCALE))
```